.circleci/config.yml

```diff
@@ -7,24 +7,28 @@ workflows:
     jobs:
       - architect/push-to-app-catalog:
           context: "architect"
-          executor: "app-build-suite"
+          # app-build-suite breaks on "application.giantswarm.io/team" in _helpers test
+          #executor: "app-build-suite"
           name: app-catalog
           app_catalog: "giantswarm-catalog"
           app_catalog_test: "giantswarm-test-catalog"
           chart: "grafana"
           persist_chart_archive: true
+          ct_config: ".circleci/ct-config.yaml"
           # Trigger job on git tag.
           filters:
             tags:
               only: /^v.*/
 
       - architect/push-to-app-catalog:
           context: "architect"
-          executor: "app-build-suite"
+          # app-build-suite breaks on "application.giantswarm.io/team" in _helpers test
+          #executor: "app-build-suite"
```

Comment:
I'd like it if we added this tbh

Reply:
Requires changes in app-build-suite. Today, all our "chart-depending" apps have had to remove this.
I agree though, would be good to update app-build-suite now we know it's the way we're going with our apps.

Reply:
I added it to the prometheus-operator-app without issues though. Why do we need to remove it?